include/revng/ABI/Definition.h

```diff
@@ -310,6 +311,27 @@ class Definition : public generated::Definition {
   bool
   checkCompatibility(const SortedVector<ArgReg> &ArgumentRegisters,
                      const SortedVector<RVReg> &ReturnValueRegisters) const;
+
+  /// \brief Compute the natural alignment of the type in accordance with
```

Comment:
Did one more slipped through? 😅. Sorry.

`<excuse>` `vscode` is automatically putting those in. I'm doing my best to remove them, but sometimes they somehow still get their way in :) `</excuse>`

Reply:
We should `sed` them out of the whole codebase and ban them from `revng-check-conventions`.
Not today though.

Reply:
About vscode: check the `doxdocgen.generic.briefTemplate` configuration, maybe it fixes your problem.

Reply:
Ah, wait, it was a custom extension all this time 😅.

I had no idea. It was probably installed implicitly by "install this to highlight syntax in `doxygen` config files or something". My bad.

Reply:
Ok, this is fixed (luckily, this was the only one of mine, but I also found a couple of these across the codebase, so there's 
`Stop using \brief doxygen command` commit removing them. Please review.

Reply:
Ah nice! So we're `\brief` free? If so, please announce on ~psa that `\brief` is banned from `revng` codebase and that once `revng-c` will comply too it will end up in `check-conventions`.

Reply:
There's still `\brief`s at the start of cpp files. I didn't touch those because that's kind of a convention. But once we decide what to do with those, why not :tada:

Reply:
> please announce on `~psa`

Once the commit goes in? Or now? Should I make a separate PR for it?

Reply:
Once it goes in. Let's scare people out of it, we'll do the final mile later.

> There's still \briefs at the start of cpp files

Not sure if we drop that the doc is affected. Honestly we don't look at the quality of the generated doxygen that much.
But let's stay focused and just scary people out of `\brief`. The thing at the beginning of the file can be fixed with `sed`.

Reply:
> and that once `revng-c` will comply too

I forgot to mention that it already complies. When searching for them, I ran the search there as well, but found nothing (since it also uses a different license header).

Reply:
P. S. I'm closing this thread.